News: GRASS GIS 8.3.2 released (#413)
OSGeo/grass-website: content/news/2024_03_07_GRASS_GIS_8_3_2_released.md

---
title: "GRASS GIS 8.3.2 released"
date: 2024-03-07T10:42:00+01:00
layout: "news"
author: Markus Neteler and the GRASS Development Team
---

### What's new in a nutshell

The GRASS GIS 8.3.2 maintenance release contains more than **30 changes**
compared to 8.3.1. This new patch release includes
**important fixes and improvements** to the GRASS GIS modules and the
graphical user interface (GUI), making it even more stable for daily work.

> Some of the most important changes are: fixes for `r.horizon` and some
> other modules; also database connections in vector data have been improved
> as well as selected manual pages.

Reviewer: ```suggestion
Some of the most important changes are: fixes for `r.horizon` and 
other modules; improvements in database connections for vector data 
as well as selected manual pages.
```

Translations continue in
[Weblate](https://weblate.osgeo.org/projects/grass-gis/), which
automatically creates pull requests with the translated chunks.
We'd like to **thank the translators of all languages** for their
ongoing support!

<a href="/images/news/grass832_TODO.png">

>   <img src="/images/news/grass832_TODO.png"

Reviewer: What about this one that @cwhite911 made for a workshop last saturday? 
https://github.com/tomorrownow/intro-to-geoprocessing-workshop/blob/main/images/lc_spatial_scale.png

Author: (1a09f8f)
